annotations/fenix/metrics/preferences.telemetry/README.md

```diff
@@ -0,0 +1,7 @@
+---
+labels:
+  - Telemetry
+---
+
+This metric has been replaced by `preferences.telemetry_enabled` and should not be used.
```

Comment:
This is meta to whether or not this is a good annotation, but is this something we would ever see any value other than true (and not feel like something was wrong...)

Reply:
I believe this is in the metric description already, so I think we're good.